packages/std/src/math/uint256.rs

```diff
@@ -191,6 +208,10 @@ impl Uint256 {
     pub fn saturating_mul(self, other: Self) -> Self {
         Self(self.0.saturating_mul(other.0))
     }
+
+    pub(crate) const fn to_words(self) -> [u64; 4] {
+        self.0 .0
```

Comment:
```suggestion
        self.0.0
```

Reply:
Cargo fmt insisted on that change for me for some reason

Reply:
https://github.com/rust-lang/rustfmt/issues/4779 - this seems related

Reply:
Nice find. I'd suggest using version "Two" if I wasn't confident we can get rid of `to_words` relatively easy.

Closing here.